cookietemple/create/templates/web/website_python/flask/{{ cookiecutter.project_slug_no_hyphen }}/deployment_scripts/setup.sh

```diff
@@ -1,29 +1,42 @@
 #!/bin/bash
 # Reference:
 # https://www.digitalocean.com/community/tutorials/how-to-serve-flask-applications-with-gunicorn-and-nginx-on-ubuntu-18-04
+sudo apt-get install software-properties-common
+sudo apt-add-repository universe
+sudo apt-get update
+sudo apt-get install python3-pip
+sudo apt-get install python3-dev nginx -y
 
-cp /home/{{cookiecutter.vmusername}}/{{cookiecutter.project_slug}}/deployment_scripts/{{cookiecutter.project_slug_no_hyphen}}.service \
+sudo pip3 install virtualenv
+virtualenv dpenv
+source dpenv/bin/activate
+
+sudo apt-get update
+pip3 install gunicorn
+make install
+
+sudo cp /home/{{cookiecutter.vmusername}}/{{cookiecutter.project_slug}}/deployment_scripts/{{cookiecutter.project_slug_no_hyphen}}.service \
```

Comment:
Does this require sudo?

Reply:
May not, but didn't have the time to test it, will look at it when deploying my website